conf/deployment/aws/gp2_csi_storage_class.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config is for AWS deployment with gp2 CSI storage class provisioning
+DEPLOYMENT:
+  gp2_csi: True
```

Comment:
thinking maybe it would be better to call the new key `customized_deployment_storage_class` and set it to the name of the actual storage class, so in this case - `gp2-csi`

Reply:
Yes, sounds reasonable.

Reply:
Will I rename the file to `gp2_csi_customized_deployment_storage_class.yaml`?